bindings/python/libmata/nfa/strings.pyx

```diff
@@ -0,0 +1,138 @@
+cimport libmata.nfa.nfa as mata
+cimport libmata.nfa.strings as mata_strings
```

Comment:
Exactly as you do here with `mata_strings`, I think that using `mata_nfa` or simply `nfa` and `strings` for import aliases would be great. `mata` for NFAs is suboptimal and not very clear in what it actually imports.

Reply:
I agree. The previous solution was least effort so the binding compiles. It leads to `mata_nfa.Nfa.*` though, which is not so pretty, so we might consider "unmaking" the class functions.

Reply:
That is true, but it corresponds to what we have in C++ Mata: `Mata::Nfa::Nfa::foo()`. I would use either `mata_nfa` or only `nfa` and keep the current nested structure as it is, personally.

What would the desired result with the "unmade" class functions look like?

Reply:
Instead of `mata_nfa.Nfa.union(...)` one would have `mata_nfa.union(...)`, the function would be normal functions and not class function. I believe I made them class function, since it was single module `libmata`, which could contain both `Nfa` and `Afa`. But now they probably miss the previous point` and could be made as normal functions.

Reply:
Not in this PR, though.

Reply:
Oh, sorry. I forgot that all the functions in binding are now implemented as class methods. You are right. They should be definitely made into normal function in module `mata.nfa`. Therefore, we would call them with the import of `mata_nfa` as `mata_nfa.union()`, as you show. That should be done. I will take a note, and we will resolve this issue in another PR.

Reply:
I will add this to my todo as well.